<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HDF5MaskLoader_Widget</class>
 <widget class="QWidget" name="HDF5MaskLoader_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="load_single_hdf5_mask_button">
     <property name="text">
      <string>Load Single HDF5 Mask</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="load_multi_hdf5_mask_button">
       <property name="text">
        <string>Load Multiple HDF5 Masks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="multi_hdf5_name_pattern_text">
       <property name="placeholderText">
        <string>*.h5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_pattern">
       <property name="text">
        <string>Filename Pattern</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 